contrib/julia-mode.el

```diff
@@ -582,12 +591,12 @@ foo() =
     (julia--should-indent
      "
 if foo
-        
+
 bar
 end"
      "
 if foo
-    
+
```

Comment:
Sorry, I actually checked for this, but I must have missed it, when I amended the commits. My whitespace-removal trigger is messing this up.